scripts/integration/docker-compose.aws.yml

```diff
@@ -1,3 +1,5 @@
+version: "3"
```

Comment:
Just for my own education, what prompted this?

Reply:
If there's no version field, it defaults to the oldest possible version (almost like Rust's editions) and a lot of the features being used in the compose file don't work on the oldest version.

Really, all of the integration compose files should have this, so I'll file a bug... but I was just focused on getting it running locally.

Reply:
Makes sense. I've never used docker compose so TIL.